### ISSUE TITLE
Additional harvest field names

### ISSUE THREAD

**Issue by gfrc11752** [Huh/PopR_SDGFP#21]
Please include hunter success, tag success, satisfaction rating, harvest density (needs to be calculated), and tag density (needs to be calculated). I am sure I will think of more down the road but that is all for now.....

**Comment by Huh** [comment]
For tag success do you want first, second and third columns?

**Comment by gfrc11752** [comment]
Ur thoughts before mine???

**Comment by gfrc11752** [comment]
I suppose, not sure how else to look at it even though harvest effort is drastically different between a “any” vs. “antlerless” tag,

**Comment by Huh** [comment]
I am not sure it means a hell of a lot to me, so I might take just the first column.  I could also see it being more useful to just convert all of this into the tag potential type metric because that means something to me.
When I say it does not mean much to me it is in part because the licensing and tag structure is so complicated that I can't mentally come up with quick and easy metrics to put the information on the ground.  The hunter success means something to me in very real terms, but at the tag level maybe guys prefer to prioritize a buck tag ahead of a doe tag or for me it is the other way around and without knowing something about the behavior of the hunters I don't know what to think about the tag success percent.  Comparing across tags is tough too.  For example, given my preference I am more likely to eat tag soup on any tag that is either sex or if I only have one tag.  If I have multiple tags then I will look like a killing machine.  
I just don't know what to make of it, but if I were you I would think about how the information is used and then we should summarize or present it in a fashion that helps that process.  
 


     On Thursday, August 6, 2015 12:52 PM, gfrc11752 <notifications@github.com> wrote:
   

 Ur thoughts before mine???

**Comment by Huh** [comment]
Agreed and it matters in calculating certain quantities, but at a glance I can't make much sense of it....but I am not a manager either. Josh Nowak 


     On Thursday, August 6, 2015 1:00 PM, Josh Nowak <Joshnwk@yahoo.com> wrote:
   

 I am not sure it means a hell of a lot to me, so I might take just the first column.  I could also see it being more useful to just convert all of this into the tag potential type metric because that means something to me.
When I say it does not mean much to me it is in part because the licensing and tag structure is so complicated that I can't mentally come up with quick and easy metrics to put the information on the ground.  The hunter success means something to me in very real terms, but at the tag level maybe guys prefer to prioritize a buck tag ahead of a doe tag or for me it is the other way around and without knowing something about the behavior of the hunters I don't know what to think about the tag success percent.  Comparing across tags is tough too.  For example, given my preference I am more likely to eat tag soup on any tag that is either sex or if I only have one tag.  If I have multiple tags then I will look like a killing machine.  
I just don't know what to make of it, but if I were you I would think about how the information is used and then we should summarize or present it in a fashion that helps that process.  
 


     On Thursday, August 6, 2015 12:52 PM, gfrc11752 <notifications@github.com> wrote:
   

 Ur thoughts before mine???

**Comment by gfrc11752** [comment]
During years of “too many” deer the managers wanted an idea of how many deer were actually be harvested of the third tag, which was free. Some tried to make the correlation if third tag success was going down there must be less deer out there to shoot so it was working. That about made me lose my mind….

**Comment by Huh** [comment]
I see.  Well, I can put them in there so you can have the conversation about what they mean or we can omit them and argue that tag allocation should rely on the IPM or other methods that attempt to quantify meaningful relationships.  
What makes a tag a third tag anyway?  If I have two tags that I fill on does and then a third buck I would fill the freezer with the doe tags and then sit on the buck tag waiting on a 'booner.  In that case, the third tag is maybe correlated with mature bucks in the area, but little else.  I see this is where you lost your mind....got it. 


     On Thursday, August 6, 2015 1:09 PM, gfrc11752 <notifications@github.com> wrote:
   

 During years of “too many” deer the managers wanted an idea of how many deer were actually be harvested of the third tag, which was free. Some tried to make the correlation if third tag success was going down there must be less deer out there to shoot so it was working. That about made me lose my mind….
